bindata/manifests/cni-config/sriov-cni-config.yaml

```diff
@@ -33,6 +34,14 @@ spec:
 {{- end -}}
 {{- if .StateConfigured -}}
   "link_state":"{{.SriovCniState}}",
+{{- end -}}
+{{- if and (.RdmaIsolation) (eq .CniType "ib-sriov") -}}
+  "rdmaIsolation": true,
 {{- end -}}
   {{.SriovCniIpam}}
-}'
+{{- if and (.RdmaIsolation) (eq .CniType "sriov") -}}
+  },
+  {"type": "rdma"
```

Comment:
@zshi-redhat will we ship rdma-cni in OCP?

Reply:
not planned in 4.7.